sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -782,17 +756,15 @@ def _from_normal_time(self, value):
     """Convert signed to "lexicographically ordered unsigned"."""
     return value + _TIME_SHIFT
 
-  def encode_to_stream(self, value, out, nested):
-    # type: (IntervalWindow, create_OutputStream, bool) -> None
+  def encode_to_stream(self, value: IntervalWindow, out: create_OutputStream, nested: bool) -> None:
```

Comment:
This looks wrong...

Reply:
The annotation is consistent with the original comment, what looks wrong?

Reply:
create_OutputStream is not (necessarily) a type. I think it's possible that the comment was wrong. (Either that or perhaps create_OutputStream should be renamed to OutputStreamType.)